app/models/person.rb

```diff
@@ -898,14 +898,24 @@ def set_ridp_for_paper_application(session_var)
     user.ridp_by_paper_application
   end
 
+  def attributes_changed?
+    person_changes = meaningful_changes(changed_attributes)
+    embedded_changes = embedded_attributes_changed?
+
+    person_changes || embedded_changes
+  end
+
   private
 
-  def attributes_changed?
-    changes = changed_attributes.stringify_keys
+  def meaningful_changes(changes)
```

Comment:
What do you think about changing to something with a question mark since this method returns a boolean result?
Maybe something like `has_meaningful_changes?(changes)` or `meaningful_changed?(changes)`